pandas/tools/plotting.py

```diff
@@ -1574,6 +1577,31 @@ def _post_plot_logic(self, ax, data):
         ax.set_ylabel(pprint_thing(y))
         ax.set_xlabel(pprint_thing(x))
 
+    def _compute_plot_data_with_dt(self):
```

Comment:
If this is only called in `_compute_plot_data` in ScatterPlot, why not just putting the code in that function?